public/app/features/dashboard-scene/settings/annotations/AnnotationSettingsEdit.tsx

```diff
@@ -0,0 +1,317 @@
+            value={annotation.enable}
+            onChange={onChange}
+            data-testid={selectors.pages.Dashboard.Settings.Annotations.NewAnnotation.enable}
+          />
+        </Field>
+        <Field
+          label="Hidden"
+          description="Annotation queries can be toggled on or off at the top of the dashboard. With this option checked this toggle will be hidden."
+        >
+          <Checkbox
+            name="hide"
+            id="hide"
+            value={annotation.hide}
+            onChange={onChange}
+            data-testid={selectors.pages.Dashboard.Settings.Annotations.NewAnnotation.hide}
+          />
+        </Field>
+        <Field label="Color" description="Color to use for the annotation event markers">
+          <HorizontalGroup>
+            <ColorValueEditor value={annotation?.iconColor} onChange={onColorChange} />
+          </HorizontalGroup>
+        </Field>
+        <Field label="Show in" data-testid={selectors.pages.Dashboard.Settings.Annotations.NewAnnotation.showInLabel}>
+          <>
+            <Select
+              options={panelFilters}
+              value={panelFilter}
+              onChange={onFilterTypeChange}
+              data-testid={selectors.components.Annotations.annotationsTypeInput}
+            />
+            {panelFilter !== PanelFilterType.AllPanels && (
+              <MultiSelect
+                options={selectablePanels}
+                value={selectablePanels.filter((panel) => annotation.filter?.ids.includes(panel.value!))}
+                onChange={onAddFilterPanelID}
+                isClearable={true}
+                placeholder="Choose panels"
+                width={100}
+                closeMenuOnSelect={false}
+                className={styles.select}
+                data-testid={selectors.components.Annotations.annotationsChoosePanelInput}
+              />
+            )}
+          </>
+        </Field>
+      </FieldSet>
+      <FieldSet>
+        <h3 className="page-heading">Query</h3>
+        {ds?.annotations && dsi && (
+          <StandardAnnotationQueryEditor
+            datasource={ds}
+            datasourceInstanceSettings={dsi}
+            annotation={annotation}
+            onChange={(annotation) => onUpdate(annotation, editIndex)}
+          />
+        )}
+        {ds && !ds.annotations && (
+          <AngularEditorLoader
+            datasource={ds}
+            annotation={annotation}
+            onChange={(annotation) => onUpdate(annotation, editIndex)}
+          />
+        )}
+      </FieldSet>
+      <Stack>
+        {!annotation.builtIn && (
+          <Button
+            variant="destructive"
+            onClick={onDeleteAndLeavePage}
+            data-testid={selectors.pages.Dashboard.Settings.Annotations.NewAnnotation.delete}
+          >
+            Delete
+          </Button>
+        )}
+        <Button onClick={onBackToList} data-testid={selectors.pages.Dashboard.Settings.Annotations.NewAnnotation.apply}>
```

Comment:
```suggestion
        <Button variant="secondary" onClick={onBackToList} data-testid={selectors.pages.Dashboard.Settings.Annotations.NewAnnotation.apply}>
```
![Captura de pantalla 2024-02-12 a las 12 54 16](https://github.com/grafana/grafana/assets/5699976/aba0f1a5-e7c3-43fd-8252-0cba02a0854d)
By default, the variant is primary, but for back to list we're using secondary :)